src/nix/doctor.cc

```diff
@@ -43,20 +42,26 @@ struct CmdDoctor : StoreCommand
         }
         success &= checkStoreProtocol(store->getProtocol());
 
-        if (!success)
+        if (success)
+            std::cout << "INFO: All checks passed!" << std::endl;
+        else {
+            std::cout << "WARNING: Not all checks passed!" << std::endl;
             throw Exit(2);
+        }
     }
 
     bool checkNixInPath()
     {
+        std::cout << "INFO: Checking that there is only one nix on PATH." << std::endl;
```

Comment:
While this is useful information it might be better to only log the steps on debug level instead.

Reply:
Logically this feels like a test suite, which generally print all the tests run with their status. Particularly with the colorized output now, there's something to be said for the warm fuzzy feeling of seeing a bunch of green PASSes scroll by :) 

Since the tests that we run are conditional on the system config, as well, I think it's reasonable to print them all.  Maybe if someday we have dozens of checks it might be worth reconsidering, but as of now we only have 3.